kibana.md

```diff
@@ -157,6 +156,35 @@ When the ES plugin boots up, it follows some logic to evaluate where to read the
 
 ![config loading diagram](ror_config_loading_diagram.png)
 
+### Malformed configuration index 
+When configuration index contains invalid configuration format, 
```

Comment:
If for some reason the in-index settings get corrupted and ROR can't parse them,